swan-lake/learn-the-platform/run-in-the-cloud/function-as-a-service/azure-functions.md

```diff
@@ -194,19 +185,25 @@ Now, as all the infrastructure required are up and running and configured, start
 import ballerina/log;
```

Comment:
```suggestion
import ballerina/log;
```
Shall we add this code inside a `ballerina` section?

Reply:
It is inside a ballerina code block